spells.yml

```diff
@@ -3518,6 +3656,30 @@ erupting_earth:
     - sorcerer
     - transmutation
     - wizard
+eruption:
+  casting_time: 1 standard action
+  components: S
+  concentration: true
+  description: |-
+    You cause a fountain of lava to burst from the ground at a point you can see within range. Each creature within a 40-foot-radius sphere centered on the point you chose must make a Dexterity saving throw. A target takes 4d12 bludgeoning damage and 4d12 fire damage on a failed save, or half as much on a success.  This spell damages objects in the area and ignites flammable objects that aren't being worn or carried.
+
+    The ground in the spell's area becomes a lake of molten lava for the duration, which is difficult terrain. A creature that ends its turn in the lava takes 4d12 fire damage, and a creature that ends its turn within 10 feet of the lava (but not in it) takes 2d12 fire damage.
+
+    On each of your subsequent turns for the duration of the spell, you can use a standard action to cause debris to spew from the lava lake. Roll a d4; the result is how many lava bombs are hurled from the area, as if you had cast _lava bombs_. Each lava bomb lands in a point you choose within 60 feet of the lava lake.
+
```

Comment:
I'm going to bring up what I mentioned last time - this is an absolutely nuts amount of damage. Comparing it to each other 8th level spell makes this pretty clear:

- _Dessication_: 12d8 (54) damage, moderate area
- _Incendiary Cloud_: 10d8 (45) ongoing damage, moderate area
- _Sunburst_: 12d6 (42) damage immediately, massive area
- _Tsunami_: 6d10 (35) ongoing, albeit decreasing, damage, massive area

And, for comparison:

- _Eruption_: 8d12 (52) damage, 4d12 (26) ongoing damage, additional ongoing damage if using the _lava bomb_s, very large area

It's just WAY too much damage for the size and ongoing effects that it has. I'd suggest turning the damage down to maybe 6d12 upfront and 3d12 ongoing, at least for a start, or possibly changing the damage dice.

Reply:
Nerfed it as you suggested. It feels very d12-y to me, so I don't want to change that. Of course, it's still probably the best 8th-level damage spell, but I think that's fine as a sorcerer exclusive.